.github/workflows/ci.yaml

```diff
@@ -123,37 +68,34 @@ jobs:
         run: |
           ./e2e/tests.sh
 
-  package:
-    runs-on: ubuntu-20.04
-    needs: e2e-test
-    steps:
-      - name: "Checkout"
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-
-      - name: "Restore the wasm files cache"
-        uses: actions/cache@v2
-        with:
-          path: |
-            build/main.wasm
-          key: ${{ runner.os }}-cache-build-${{ github.sha }}
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Login to DockerHub
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ghcr.io/${{ github.repository }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}
+
+      - name: Login to GHCR
```

Comment:
Publishing to GHCR since it's easiest for OSS projects. Forks work as-is without any issues with secrets